packages/terra-disclosure-manager/src/terra-dev-site/doc/disclosure-manager/DisclosingContentWithAHeader.4.doc.mdx

```diff
@@ -0,0 +1,77 @@
+import { Badge } from 'terra-disclosure-manager/package.json?dev-site-package';
+
+<Badge />
+
+# Disclosing Content With a Header
+Implementations of the DisclosureManager must render a header containing controls for the various disclosure management actions (close, back, maximize/minimize, etc.). 
+
+The `DisclosureManagerHeaderAdapter` can be used to register header information to the disclosing manager. This can be usd to register a title and a collapsibleMenuView.
+
+## DisclosureManagerHeaderAdapter
+
+The DisclosureManagerHeaderAdapter can be rendered by the disclosed content to inject their own component-specific contents into that header.
+
+`DisclosureManagerHeaderAdapter` Props:
+
+|Prop|Is Required|Description|
+|---|---|---|
+|`title`|optional|A string to render as the header's title.|
+|`collapsibleMenuView`|optional|A CollapsibleMenuView component to render within the header.|
+
+```jsx
+import { DisclosureManagerContext, DisclosureManagerHeaderAdapter } from 'terra-disclosure-manager';
+import Button from 'terra-button';
+import CollapsibleMenuView from 'terra-collapsible-menu-view';
+
+const DisclosedComponent = () => {
+  const disclosureManager = React.useContext(DisclosureManagerContext);
+
+  return (
+    <React.Fragment>
+     <DisclosureManagerHeaderAdapter
+       title="My Disclosure Component"
+       collapsibleMenuView={<CollapsibleMenuView />}
+     />
+      <Button
+        text="Close Modal"
+        onClick={() => {
+          disclosureManager.closeDisclosure();
+        }}
+      />
+    </React.Fragment>
+  );
+};
+```
+
+This will register the header information on mount.
+
+## Providing a Title on Disclose
+It has been observed that there can be a 'pop-in' affect with the header render in slow domains and using the `DisclosureManagerHeaderAdapter`. An option to 
```

Comment:
Did we decide we weren't going to do this? Or is that still up in the air pending the other performance investigations.

Reply:
@tbiethman pending investigations. I will remove from this PR and add to the other PR if these changes get it.